kythe/go/serving/tools/http_server/http_server.go

```diff
@@ -45,7 +45,7 @@ import (
 var (
 	servingTable = flag.String("serving_table", "", "LevelDB serving table")
 
-	httpListeningAddr = flag.String("listen", "localhost:8080", "Listening address for HTTP server")
+	httpListeningAddr = flag.String("listen", "localhost:8080", "Listening address for HTTP server (":<port>" allows access from any machine)")
```

Comment:
The embedded quotes need to be escaped.

Reply:
Fixed. (I must have rebuilt everything in the wrong branch ...)